vivisect/analysis/arm/emulation.py

```diff
@@ -175,8 +182,33 @@ def buildFunctionApi(vw, fva, emu, emumon):
     return api
 
 
+def getAllReads(emu):
+    # there has to be a more efficient way...
+    allaccess = []
+    for path in vg_path.getAllPaths(emu.path):
+        for node in path:
+            for read in node[2]['readlog']:
+                if read in allaccess:
+                    # skip duplicate
+                    continue
+                allaccess.append(read)
```

Comment:
To save some memory pressure, we could turn these into yield-ers

Reply:
done